packages/components/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Internal
+-   `SlotFill`: Refactor `<Slot bubblesVirtually />` ([#53272](https://github.com/WordPress/gutenberg/pull/53272))
```

Comment:
As a changelog entry, "refactor" doesn't say anything meaningful. Let's describe the user-visible change in `<Slot bubblesVirtually>`: when rendered with `children`, it will ignore them. Previously they would be rendered.